TestModels/aws-sdks/ddb/Model/model.json

```diff
@@ -3841,9 +3841,7 @@
         },
         "aws.protocols#awsJson1_0": {},
         "smithy.api#documentation": "<fullname>Amazon DynamoDB</fullname>\n         <p>Amazon DynamoDB is a fully managed NoSQL database service that provides fast\n            and predictable performance with seamless scalability. DynamoDB lets you\n            offload the administrative burdens of operating and scaling a distributed database, so\n            that you don't have to worry about hardware provisioning, setup and configuration,\n            replication, software patching, or cluster scaling.</p>\n         <p>With DynamoDB, you can create database tables that can store and retrieve\n            any amount of data, and serve any level of request traffic. You can scale up or scale\n            down your tables' throughput capacity without downtime or performance degradation, and\n            use the Amazon Web Services Management Console to monitor resource utilization and performance\n            metrics.</p>\n         <p>DynamoDB automatically spreads the data and traffic for your tables over\n            a sufficient number of servers to handle your throughput and storage requirements, while\n            maintaining consistent and fast performance. All of your data is stored on solid state\n            disks (SSDs) and automatically replicated across multiple Availability Zones in an\n                Amazon Web Services Region, providing built-in high availability and data\n            durability.</p>",
-        "smithy.api#suppress": [
-          "RuleSetAwsBuiltIn.AWS::Auth::AccountId"
-        ],
+        "smithy.api#suppress": ["RuleSetAwsBuiltIn.AWS::Auth::AccountId"],
```

Comment:
Why is this in the diff?

Reply:
Its result of make format_java

Reply:
I will revert it back for now. This PR has gotten very long. We should figure it out in other PR.

Reply:
Reverted it